x11-plugins/gkrellm-cpupower/gkrellm-cpupower-0.2-r2.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gkrellm-plugin toolchain-funcs
+
+MY_P="${P/gkrellm/gkrellm2}"
+
+DESCRIPTION="A Gkrellm2 plugin for displaying and manipulating CPU frequency"
+HOMEPAGE="https://github.com/sainsaar/gkrellm2-cpupower/"
+SRC_URI="https://github.com/sainsaar/gkrellm2-cpupower/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="app-admin/gkrellm:2[X]"
+RDEPEND="
+	${DEPEND}
+	app-admin/sudo
+	sys-power/cpupower"
+
+S="${WORKDIR}/${MY_P}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-makefile.patch
+)
+
+src_install() {
```

Comment:
Change to match [phase execution order](https://devmanual.gentoo.org/ebuild-writing/functions/index.html)

Reply:
Done. Silly me.